assets/js/googlesitekit/modules/datastore/modules.test.js

```diff
@@ -420,6 +420,39 @@ describe( 'core/modules modules', () => {
 				).toMatchObject( { ...state, [ slug ]: true } );
 			} );
 		} );
+
+		describe( 'receiveCheckModuleAccess', () => {
```

Comment:
We don't need tests for the `receiveCheckModuleAccess` because it is a generated method and tested in the `createFetchStore` tests.

Reply:
@eugene-manuilov, we should add tests as per the AC:
>Test coverage should be added for the new selector and receiveCheckModuleAccess action created by the fetch store

Reply:
That's strange. We don't test `receive*` selectors as far as I remember. @tofumatt or @aaemnnosttv can you confirm that we really need it?

Reply:
Yes, we do test `receiveCheckRequirementsError` and `receiveCheckRequirementsSuccess` actions generated by `createFetchStore` in the same `assets/js/googlesitekit/modules/datastore/modules.test.js` file.

Reply:
It's a bit strange from my point of view because that functionality is already tested. Anyway, let's leave those tests for now then. If Matthew or Evan also think that we don't need it, then we will get rid of it.

Reply:
Yes, I understand your point that it's been already tested in `createFetchStore`.